nbextensions/usability/hinterland/hinterland.js

```diff
@@ -69,8 +70,13 @@ define(function (require, exports, module) {
 						}, cur);
 
 						if (pre_cursor !== '' && !ignore_re.test(pre_cursor)) {
-							completer.startCompletion();
-							completer.autopick = false;
+							if (pre_cursor === '(') {
+								cell.tooltip.request(cell);
```

Comment:
ah, gotcha! I hadn't really appreciated what you meant. Mm, although certainly possible, this would involve digging into and patching the jupyter internals of how the tooltips are displayed, so it may take a little longer...